src/libs/notificationService.js

```diff
@@ -19,6 +19,18 @@ export const NotificationService = {
     });
   },
 
+  /**
+   * Get the raw banner content from GCS
+   * @returns {Promise<JSON>}
+   */
+  getActiveBanners: async () => {
+    const banners = await NotificationService.getBanners();
+    if (!fp.isEmpty(banners)) {
+      return fp.find({active: true})(banners);
```

Comment:
find() will only return the first element that is true, which can be an issue if there are multiple messages that are active. Use something like map() to create a new array of elements that should be included.

Reply:
good catch!! thank you!